### ISSUE TITLE
Document the need for journald flush to be persistent

### ISSUE THREAD

**Issue by lal12** [systemd/systemd#19480]
I had some issues making the journal on my embedded system persistent. Took me a while to find out that apparently (ea6838c9f0a447e0807369368f7800770f9f1dea) just writing `Storage=persistent` in `journald.conf` is not enough, but calling `journalctl --flush` or sending `SIGUSR1` to journald is needed too.

I guess this at least should be mentioned below the "Storage=persistent" entry on the manpage.

**Comment by poettering** [comment]
We automatically run --flush at boot. So this should happen automatically?

**Comment by lal12** [comment]
Yeah the `systemd-journal-flush.service` does, though I have sorted out a lot of default systemd services, since most aren't needed on our embedded system. In this case I sorted out one too much. 

But I don't think that "this is done by default" excludes this from automatically being excluded from the man page. In other cases special services have a man page of their own `systemd-journal-flush.service` does not.
